backend/open_webui/routers/retrieval.py

```diff
@@ -347,6 +348,7 @@ async def get_rag_config(request: Request, user=Depends(get_admin_user)):
     return {
         "status": True,
         "pdf_extract_images": request.app.state.config.PDF_EXTRACT_IMAGES,
+        "RAG_FULL_CONTEXT": request.app.state.config.RAG_FULL_CONTEXT,
```

Comment:
I saw this in the original PR and and destroys me that it doesn't follow convention.